tests/_runner.lua

```diff
@@ -32,7 +32,7 @@ end)
 runner.run_steps = function(steps)
```

Comment:
Commit message (ff4ad53af8): s/we are done to 60 seconds/we are down to 60 seconds/